functions/Add-DbaAgReplica.ps1

```diff
@@ -110,6 +122,7 @@ function Add-DbaAgReplica {
         [ValidateSet('Automatic', 'Manual')]
         [string]$SeedingMode,
         [string]$Endpoint,
+        [string]$EndpointUrl,
```

Comment:
No, they are not. Because of `-ReadonlyRoutingConnectionUrl` this command can only add one Replica at a time - because this Url aswell must be different for every replica.

Reply:
https://github.com/sqlcollaborative/dbatools/blob/475158e77caee1c5589ca952bad663f2c7aba4ed/functions/Add-DbaAgReplica.ps1#L98

Reply:
Yes, this has to be changed to `[DbaInstanceParameter]$SqlInstance,`.